beetsplug/play.py

```diff
@@ -90,6 +90,9 @@ def play_music(self, lib, opts, args):
                 command_str = command_str.replace(ARGS_MARKER, opts.args)
             else:
                 command_str = u"{} {}".format(command_str, opts.args)
+        else:
+            if ARGS_MARKER in command_str:
```

Comment:
This condition shouldn't be necessary. `.replace` will just silently leave the string unchanged if it doesn't contain the "needle."